certif/app/controllers/authenticated/team/invite.js

```diff
@@ -2,19 +2,90 @@ import Controller from '@ember/controller';
 import { service } from '@ember/service';
 import { action } from '@ember/object';
 
+import ENV from 'pix-certif/config/environment';
+import get from 'lodash/get';
+
 export default class AuthenticatedTeamInviteController extends Controller {
+  @service currentUser;
+  @service intl;
+  @service notifications;
   @service router;
+  @service store;
+
+  email = null;
 
   @action
   async createCertificationCenterInvitation(event) {
     event.preventDefault();
+
+    const certificationCenterId = this.currentUser.currentAllowedCertificationCenterAccess.id;
+    const emailsTextareaInput = Object.values(event.target)
+      .filter((target) => target.type === 'textarea')
+      .shift();
+    const emails = emailsTextareaInput.value.replace(/[\s\r\n]/g, '').split(',');
+
+    try {
+      await this.store.adapterFor('certification-center-invitation').sendInvitations({ certificationCenterId, emails });
+
+      const message =
+        emails.length > 1
+          ? this.intl.t('pages.team-invite.notifications.success.invitations')
+          : this.intl.t('pages.team-invite.notifications.success.invitation', { email: emails.shift() });
```

Comment:
C'est juste une petite remarque sans grande importance : il me semble que la fonction `shift()` est plus adaptée dans le cas de boucle où il y a plusieurs shift à réaliser, mais lorsqu'on veut récupérer la valeur unique d'une liste/tableau l'accès par index 0 avec `[0]` me semble indiquer plus clairement l'intention.

```suggestion
    const emailsTextareaInput = Object.values(event.target)
      .filter((target) => target.type === 'textarea')[0];
    const emails = emailsTextareaInput.value.replace(/[\s\r\n]/g, '').split(',');

    try {
      await this.store.adapterFor('certification-center-invitation').sendInvitations({ certificationCenterId, emails });

      const message =
        emails.length > 1
          ? this.intl.t('pages.team-invite.notifications.success.invitations')
          : this.intl.t('pages.team-invite.notifications.success.invitation', { email: emails[0] });
```